resources/europe/italy/it-discourse.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "it-discourse",
+  "type": "discourse",
+  "locationSet": {"include": ["ITA"]},
```

Comment:
```suggestion
  "locationSet": {"include": ["it"]},
```

Reply:
Yes, you're right (I'm sorry, it's my first PR on this project)

Reply:
No worries, that's what's the community is all about, helping each other :-)